### PR TITLE
docker: add Spanish translation

### PR DESCRIPTION
**Pull request by franciscollanquipichun**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/docker.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker ps`
+
+- Lista todos los contenedores de Docker (en ejecución y detenidos):
+
+`docker ps -a`
+
+- Inicia un contenedor desde una imagen con un nombre personalizado:
+
+`docker run --name {{nombre_contenedor_personalizado}} {{imagen}}`
```
Comment (navarroaxel) [inline pages.es/common/docker.md:16]:
```suggestion
`docker run --name {{nombre_de_contenedor}} {{imagen}}`
```

File: pages.es/common/docker.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- Inicia o detiene un contenedor existente:
+
+`docker {{start|stop}} {{nombre_contenedor}}`
```
Comment (navarroaxel) [inline pages.es/common/docker.md:20]:
```suggestion
`docker {{start|stop}} {{nombre_de_contenedor}}`
```

File: pages.es/common/docker.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+- Extraer una imagen de un registro de Docker:
+
+`docker pull {{imagen}}`
+
+- Inicia una shell dentro de un contenedor en ejecución:
```
Comment (navarroaxel) [inline pages.es/common/docker.md:26]:
```suggestion
- Inicia una línea de Comandos dentro de un contenedor en ejecución:
```

File: pages.es/common/docker.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker exec -it {{nombre_contenedor}} {{sh}}`
+
+- Elimina un contenedor detenido:
+
+`docker rm {{nombre_contenedor}}`
+
+- Busca y sigue los registros de un contenedor:
+
+`docker logs -f {{nombre_contenedor}}`
```
Comment (navarroaxel) [inline pages.es/common/docker.md:36]:
```suggestion
`docker logs -f {{nombre_de_contenedor}}`
```

File: pages.es/common/docker.md [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker pull {{imagen}}`
+
+- Inicia una shell dentro de un contenedor en ejecución:
+
+`docker exec -it {{nombre_contenedor}} {{sh}}`
+
+- Elimina un contenedor detenido:
+
+`docker rm {{nombre_contenedor}}`
```
Comment (navarroaxel) [inline pages.es/common/docker.md:32]:
```suggestion
`docker rm {{nombre_de_contenedor}}`
```

File: pages.es/common/docker.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+
+`docker {{start|stop}} {{nombre_contenedor}}`
+
+- Extraer una imagen de un registro de Docker:
+
+`docker pull {{imagen}}`
+
+- Inicia una shell dentro de un contenedor en ejecución:
+
+`docker exec -it {{nombre_contenedor}} {{sh}}`
```
Comment (navarroaxel) [inline pages.es/common/docker.md:28]:
```suggestion
`docker exec -it {{nombre_de_contenedor}} {{sh}}`
```

File: pages.es/common/docker.md [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+
+- Elimina un contenedor detenido:
+
+`docker rm {{nombre_contenedor}}`
+
+- Busca y sigue los registros de un contenedor:
```
Comment (navarroaxel) [inline pages.es/common/docker.md:34]:
```suggestion
- Obtiene y sigue los registros de un contenedor:
```

File: pages.es/common/docker.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+- Inicia un contenedor desde una imagen con un nombre personalizado:
+
+`docker run --name {{nombre_contenedor_personalizado}} {{imagen}}`
+
+- Inicia o detiene un contenedor existente:
+
+`docker {{start|stop}} {{nombre_contenedor}}`
+
+- Extraer una imagen de un registro de Docker:
```
Comment (navarroaxel) [inline pages.es/common/docker.md:22]: Creo que está en la traducción más acertada:
```suggestion
- Descarga una imagen desde un registro de Docker:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @franciscollanquipichun. Thanks, I left some comments.

Remember that you can send all your translations in 1 PR, just keep 1 commit per page and up to 15 pages per PR. The rule that 1 page per PR is for the primary (English) pages.

Also you can add/edit a primary page and include the Spanish translation in the same PR.

**Comment by franciscollanquipichun** [comment]
Hi @navarroaxel, thanks for the comments and suggestions

**Review by navarroaxel — APPROVED** [review]

**Review by ivanhercaz — APPROVED** [review]
Looks good to me! ¡Gracias @franciscollanquipichun!

**Pull request merged**